fastmri/data/mri_data.py

```diff
@@ -358,3 +360,111 @@ def __getitem__(self, i: int):
+
+                # if annotation (filename or slice) not found, fill in dummy values
+                if len(annotations_df) == 0:
+                    annotation = self.get_annotation(True, None)
+                    metadata['annotation'] = annotation
+                    self.annotated_examples += [(fname, slice_ind, metadata)]
+
+                else:
+                    # only use the first annotation
+                    if multiple_annotation_policy == 'first':
+                        rows = list(annotations_list)[0]
+                        annotation = self.get_annotation(False, rows)
+                        metadata['annotation'] = annotation
+                        self.annotated_examples += [(fname, slice_ind, metadata)]
+
+                    # use an annotation at random
+                    elif multiple_annotation_policy == 'random':
+                        random_number = random.randint(0, len(annotations_df) - 1)
+                        rows = list(annotations_list)[random_number]
+                        annotation = self.get_annotation(False, rows)
+                        metadata['annotation'] = annotation
+                        self.annotated_examples += [(fname, slice_ind, metadata)]
+
+                    # extend examples to have tow copies of the same slice, one for each annotation
+                    elif multiple_annotation_policy == 'all':
+                        for rows in annotations_list:
+                            annotation = self.get_annotation(False, rows)
+                            metadata['annotation'] = annotation
+                            self.annotated_examples += [(fname, slice_ind, metadata)]
+
+    def get_annotation(self, dummy_value, row):
+        if dummy_value is True:
+            annotation = {
+                'fname': "",
+                'slice': "",
+                'study_level': "",
+                'x': -1,
+                'y': -1,
+                'width': -1,
+                'height': -1,
+                'label': ""
+            }
+        else:
+            annotation = {
+                'fname': str(row.file),
+                'slice': int(row.slice),
+                'study_level': str(row.study_level),
+                'x': int(row.x),
+                'y': int(row.y),
+                'width': int(row.width),
+                'height': int(row.height),
+                'label': str(row.label)
+            }
+        return annotation
+
+    def download_csv(self, version, mri_type):
+        # request file by git hash and mri type
+        url = f'https://raw.githubusercontent.com/microsoft/fastmri-plus/{version}/Annotations/{mri_type}.csv'
+        request = requests.get(url, timeout=10, stream=True)
+        fname = f"{mri_type}{version}.csv"
+
+        # create temporary folders
+        if not os.path.isdir('.annotation_cache'):
+            os.system('mkdir .annotation_cache')
```

Comment:
Do not use system calls. Try to manage paths with `pathlib.Path`.

Reply:
Finally, I used `Path('.annotation_cache').mkdir(parents=True, exist_ok=True)` to get the same result.